File: official/docs/java/v5/pickups/cancel.java
package pickups;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;
import com.easypost.model.Pickup;

public class Cancel {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = "EASYPOST_API_KEY";

        Pickup pickup = Pickup.retrieve("pickup_...");

        pickup = pickup.cancel();

        System.out.println(pickup);
    }
}

File: official/docs/csharp/v5/pickups/cancel.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;
using EasyPost.Models.API;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            Pickup pickup = await client.Pickup.Retrieve("pickup_...");

            pickup = await client.Pickup.Cancel(pickup.Id);

            Console.WriteLine(JsonConvert.SerializeObject(pickup, Formatting.Indented));
        }
    }
}
